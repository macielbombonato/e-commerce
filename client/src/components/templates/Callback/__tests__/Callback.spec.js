import createTestContext from 'react-cosmos-test/enzyme';

import fixture from '../Callback.fixture';

const { mount, getWrapper } = createTestContext({ fixture });

beforeEach(mount);

describe('#Callback template', () =>
  it('renders component', () => {
    expect(getWrapper()).toMatchSnapshot();
  }));
