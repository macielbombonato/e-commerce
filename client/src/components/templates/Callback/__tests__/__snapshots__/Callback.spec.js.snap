// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Callback template renders component 1`] = `
.glamor-1,
[data-glamor-1] {
  position: fixed;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  justify-content: center;
  height: 100vh;
  width: 100vw;
  top: 0;
  bottom: 0;
  left: 0;
  right: 0;
  background-color: white;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
}

.glamor-0,
[data-glamor-0] {
  animation: logo-spin infinite 20s linear;
  -webkit-animation: logo-spin infinite 20s linear;
}

<Callback>
  <glamorous(div)>
    <div
      className="glamor-1"
    >
      <glamorous(img)
        alt="loading"
        src="logo.svg"
      >
        <img
          alt="loading"
          className="glamor-0"
          src="logo.svg"
        />
      </glamorous(img)>
    </div>
  </glamorous(div)>
</Callback>
`;
